app/graphql/types/base_object.rb

```diff
@@ -1,2 +1,3 @@
 class Types::BaseObject < GraphQL::Schema::Object
+  connection_type_class(Types::BaseConnection)
```

Comment:
Have you tested that this actually works?

Reply:
yeah I just retested. It allows this type of query now which is what I was aiming for. Anything that is a `connection_type` has access to this.
```
query {
  anime{
    totalCount
    nodes {
    
    }
  }
}
```